### DISCUSSION TITLE
Meeting 29.01.24

### DISCUSSION THREAD

**Discussion by ehhartmann** [graeter-group/kimmdy#358] [Ideas]
- discuss current state with kimmdy + grappa
- discuss effort to retrain HAT and timing
- TS search improvement 
- rate statistics side project; currently we could say for long sampling we get relatively correct rates?!
- (intern)

**Comment by ehhartmann** [comment]
Version issues with cuda + tensorflow + dgl + pytorch

**Reply by ehhartmann** [reply]
- TS search improvement will be used for retraining with grappa
- rate statistics side project for later -> HAT only has relative rates which are too big for now\
- examples: particle pattern formation as simplest example? or fix sd simulation
- fix install workflow for grappa, check which cuda version we should have and how tensorflow and torch are working with it.
- @KRiedmiller and @LeifSeute should agree on cuda etc. versions

Paper Skeleton:
Camilo:
- Method overview figure
- recapitulate kimmdy 1 figure
- examples figures

Journal Ideas: Nature Methods, JCTC, Living Journal of Computational Molecular Science, Journal of Open Source Software

Decide on a Meeting to write the paper skeleton

**Reply by KRiedmiller** [reply]
For now HAT and Tensorflow on GPU, Grappa on CPU
